noxfile.py

```diff
@@ -135,7 +135,7 @@ def cover(session):
     session.run("coverage", "erase")
 
 
-@nox.session(python=DEFAULT_PYTHON_VERSION)
+@nox.session(python="3.8")
```

Comment:
Does this need to be 3.8 for this to work?

Reply:
I simplified the Docker image which now only has Python 3.8. We can revert back to the old image and use Python 3.7, or install Python 3.7 then we can use the Default Python.

Reply:
I'll modify the Dockerfile so that we can use the DEFAULT_PYTHON_VERSION.

Reply:
@busunkim96 Done